Cargo.toml

```diff
@@ -137,20 +137,19 @@ flatbuffers = "23.0"
 futures-channel = "0.3"
 futures-util = { version = "0.3", default-features = false }
 getrandom = "0.2"
-glam = "0.22"                                                      # glam update blocked by macaw
+glam = "0.22" # glam update blocked by macaw
 glob = "0.3"
 gltf = "1.1"
 half = "2.3.1"
 hyper = "0.14"
 image = { version = "0.24", default-features = false }
 indent = "0.1"
-indexmap = "2.1"                                                   # Version chosen to align with other dependencies
-indicatif = "0.17.7"                                               # Progress bar
-infer = "0.15"                                                     # infer MIME type by checking the magic number signaturefer MIME type by checking the magic number signature
+indexmap = "2.1" # Version chosen to align with other dependencies
```

Comment:
my local formatter had different ideas I guess. All I wanted to do is sort some dependencies